.swiftformat

```diff
@@ -9,5 +9,6 @@
 --patternlet inline
 --stripunusedargs unnamed-only
 --ifdef no-indent
+--maxwidth 120
```

Comment:
Proposed maximum line length to make diffs more readable, especially on smaller monitors and devices. WDYT?